src/App.tsx

```diff
@@ -16,7 +16,7 @@ function App() {
           <img src={reactLogo} className="logo react" alt="React logo" />
         </a>
       </div>
-      <h1>Vite + React</h1>
+      <h1>hello world, test</h1>
```

Comment:
Good job.  Perhaps, we should consider capitalizing to make it consistent with our other messaging.  What do you think?